### DISCUSSION TITLE
[warning] [@typescript-eslint/typescript-estree] TypeScript version error

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [nuxt/nuxt#21764] [Questions]
### Warning

![image](https://github.com/nuxt/nuxt/assets/10191085/5a303ae1-fc7a-47f0-b951-df540e91d4b3)

### Attempts 

- Changed the VSCode TypeSsript version from VSCode's version to Workspace version.
- pnpm update (to update all packages)
- pnpx nuxi upgrade --force

### Environment
- Node 18.15.0
- VSCode 1.79.2
- Nuxt 3.6.0

**Comment by angelhdzmultimedia** [comment]
### Update
Downgrading from typescript 5.1.3 to 5.0.4 got rid of the warning.

**Comment by MuhammadM1998** [comment]
I think #21729 fixes this and its already merged in Nuxt 3.6.1 which is set to be released today or tomorrow.
